webserver.py

```diff
@@ -144,6 +262,20 @@ def generate(seed_id):
         return 0
 
 
+def bonus_yield(seed_id):
+    bonus = 0
+    while generate(seed_id) == 1:
+        bonus += 1
+    return bonus
+
+
+def valid_chars(slug):
+    regex = re.compile('^[a-z]+$')
+    x = regex.match(slug)
+    if x is False:
+        raise Exception("Invalid characters")
```

Comment:
We can do better than generic Exceptions now.  This seems like a BadRequest to me, but take a look at
http://werkzeug.pocoo.org/docs/0.11/exceptions/

And use an appropriate error.